python/paddle/fluid/trainer.py

```diff
@@ -48,12 +49,14 @@ class BeginStepEvent(object):
     def __init__(self, epoch_id, step_id):
         self.epoch = epoch_id
         self.step = step_id
+        self.fetch_metrics = True
```

Comment:
`fetch metrics` is slow. User can configure whether to fetch or not by `BeginStepEvent`. Fetch metrics by default.

Reply:
Curious why it's slow? Most of the case it just fetches a scalar cost.